overlord/snapstate/snapstate_update_test.go

```diff
@@ -8575,7 +8505,7 @@ func (s *snapmgrTestSuite) TestUpdateBaseKernelSingleRebootUnsupportedWithGadget
 		[]string{"kernel", "core18", "gadget"}, nil, s.user.ID, &snapstate.Flags{})
```

Comment:
same issue vs master with the additional shared lane, maybe here we want to keep as step in the right direction but I don't know how it affects spread tests or otoh is the kind of change we don't want to do without matching spread tests?